.ci/config/matrices.yaml

```diff
@@ -9,9 +9,10 @@ engine-webapp-unit:
   - 'postgresql_142'
   - 'postgresql_150'
   - 'postgresql_160'
+  - 'postgresql_170'
   - 'aws_aurora_postgresql_14'
   - 'aws_aurora_postgresql_15'
-  - 'aws_aurora_postgresql_16'
+#  - 'aws_aurora_postgresql_16'
```

Comment:
**TODO**: Figure out why this entry breaks the build

Reply:
**Root-cause**: It turned out the enablement of AWS Aurora PostgresSQL 16 requires also an entry in `jenkins-shared-library` under **databases** which was lacking from the `postgres-17-declarative-podspecs` branch.

After Rebasing the branch with latest `master` the issue dissapeared.